.maestro/autofill/4_autofill_settings_creds.yaml

```diff
@@ -0,0 +1,128 @@
+    commands:
+      - launchApp:
+          clearState: true
+      - runFlow: ../shared/skip_all_onboarding.yaml
+
+      - tapOn:
+          id: "com.duckduckgo.mobile.android:id/browserMenuImageView"
+      - assertVisible: "Settings"
+      - tapOn: "Settings"
+
+      - scrollUntilVisible:
+          element:
+            text: "Passwords.*"
+      - assertVisible: "Passwords.*"
+      - tapOn: "Passwords.*"
+
+      - assertVisible:
+          id: secondaryText
+          text: "0"
+
+      - assertVisible:
+          text: "Ask to Save and Autofill"
+
+      - assertVisible:
+          id: syncDesktopPasswordsOption
+
+      - tapOn:
+          id: passwordsListItem
+
+      - assertVisible:
+          text: "No passwords saved yet"
+      - assertNotVisible:
+          id: searchLogins
+
+      - runScript: steps/2_script.js
+
+      - repeat:
+          while:
+            true: ${output.addLogins.counter < output.addLogins.domains.length}
+          commands:
+            - tapOn:
+                id: addLoginManually
+            - assertVisible:
+                text: Add Password
+            - assertNotVisible:
+                id: view_menu_save
+
+            - scrollUntilVisible:
+                element:
+                  id: usernameEditText
+            - tapOn:
+                id: usernameEditText
+            - inputText: "user"
+
+            - assertVisible:
+                id: view_menu_save
+
+            - scrollUntilVisible:
+                element:
+                  id: passwordEditText
+            - tapOn:
+                id: passwordEditText
+            - inputText: "123"
+
+            - scrollUntilVisible:
+                element:
+                  id: domainEditText
+            - tapOn:
+                id: domainEditText
+            - inputText: "${output.addLogins.domains[output.addLogins.counter]}"
+
+            - scrollUntilVisible:
+                element:
+                  id: notesEditText
+            - tapOn:
+                id: notesEditText
+            - inputText: "a note"
+
+            - tapOn:
+                id: view_menu_save
+                retryTapIfNoChange: false
+
+            - scrollUntilVisible:
+                element:
+                  text: "Last updated.*"
+            - assertVisible: "Last updated.*"
+
+            - tapOn: "Navigate up"
+
+            - evalScript: ${output.addLogins.counter++}
+
+      - scrollUntilVisible:
+          element:
+            text: "#"
+      - assertVisible:
+          text: "#"
+
+      - scrollUntilVisible:
+          element:
+            text: "a.example.com"
+      - assertVisible:
+          text: "a.example.com"
+
+      - assertNotVisible:
+          text: "https://a.example.com"
+
+      - scrollUntilVisible:
+          element:
+            text: "fill.dev"
+      - assertVisible:
+          text: "fill.dev"
+
+      - assertNotVisible:
+          text: "fill.dev/example"
+
+      - tapOn: "Navigate up"
```

Comment:
I don't understand this addition; why is there a new test that is largely a copy/paste of the existing `2_autofill_add_search_update_delete_creds.yaml` test?

Shouldn't this new test be about testing the autofill settings screen rather than acting on the password list screen still, which is already covered?

Reply:
The test is validating the following:
- Access settings without passwords: counter is 0, options present
- We add passwords (copied from another flow)
- Go back to settings: counter is 3 (validates we show the right number)